doc/Caliptra.md

```diff
@@ -1226,20 +1249,202 @@ This section describes Caliptra error reporting and handling.
 
 Please refer to [the Caliptra code base](https://github.com/chipsalliance/caliptra-sw/blob/main/error/src/lib.rs) for a list of the error codes.
 
-# Future effort: Caliptra security subsystem
+# Caliptra Security Subsystem
+The Caliptra subsystem offers a complete RoT subsystem, with open source programmable components for customization of SoC boot flows.
 
-A future effort is a full security subsystem solution. This solution is a combination of fully open source digital logic and licensable analog components that are technology dependent, such as TRNG analog sources or technology dependent fuse controllers.
+*Figure: Caliptra security subsystem*
+![](./images/Subsystem.png)
+
+**Caliptra Subsystem Architectural Flows**
+
+**Subsystem (Pre-FW Load) Boot Flow **
+
+**Note:** Any step done by MCU HW/ROM would have been performed by “SOC Manager” in Caliptra 1p0.
+
+1. SoC (using its HW or MCU ROM) performs pre-steps like bringing up CRO or PLL, MBIST flows on SRAMs, SRAM Init etc.
+2. SOC will assert MCU & Caliptra pwrgood and after 10 cycles MCU
+3. MCU ROM or SOC Manager wrapper will bring up fuse controller and any other SOC specific infrastructure RTL modules (I3C, GPIO programming, Glitch detector programming etc.)
+4. MCU ROM or SOC Manager wrapper will deassert Caliptra reset.
+5. Caliptra HW will read the security centric (secret) fuses.
+6. MCU ROM waits for ready_for_fuses to be asserted.
+7. MCU ROM or SOC Manager will populate the remaining fuses of Caliptra and reads its own fuses (if any). Note that this step is gated behind the completion of security fuse writes to ensure the step has completed.
+8. MCU ROM will write “fuse done” to Caliptra.
+9. Caliptra will go through its boot flow of bringing up uC.
+10. Caliptra ROM starts and executes various KATs flows.
+
+**Subsystem Boot Flow**
+
+**_If (Caliptra-Passive-Mode)_**
+
+1. SOC Manager goes through legacy flows => loads Caliptra FW using legacy flows, Caliptra sets RT ready and SOC  <-> Caliptra boot flow is done.
```

Comment:
Want to emphasize @jhand2 's point that all use of the term "legacy" should be replaced with "passive"

Reply:
Aye-Aye! Taken care of...